Dockerfile

```
@@ -150,12 +150,13 @@ ENV IS_DOCKER=true
 ENV SDK_SRC=/src
 ENV OUTPUT_DIR=/out
 ENV SDK_REVISION=
+Env SDK_REPO=
 ENV SDK_BUILD=0
 
 WORKDIR /app
 COPY --chown=$USER_UID:$USER_GID . .
 
-RUN mkdir -p $SDK_SRC $OUTPUT_DIR && chown $USER_UID:$USER_GID $SDK_SRC $OUTPUT_DIR /app
+RUN mkdir -p $SDK_SRC $OUTPUT_DIR /home/node/.cache/yarn /go/pkg/mod && chown -R $USER_UID:$USER_GID $SDK_SRC $OUTPUT_DIR /home/node/.cache /go && chown $USER_UID:$USER_GID /app
```

Please split long lines.

```suggestion
RUN mkdir -p $SDK_SRC $OUTPUT_DIR /home/node/.cache/yarn /go/pkg/mod && \
  chown -R $USER_UID:$USER_GID $SDK_SRC $OUTPUT_DIR /home/node/.cache /go && \
  chown $USER_UID:$USER_GID /app
```

---

Resolved in 6287b8096ebd48bb08671feb6c24c422e917dd3f